kubernetes-tests/src/test/java/io/fabric8/openshift/client/server/mock/OpenShiftVersionInfoTest.java

```diff
@@ -136,4 +136,95 @@ void testClusterVersioningOpenshift4() {
+      "      }," +
+      "      \"status\": {" +
+      "        \"availableUpdates\": [" +
+      "          {" +
+      "            \"force\": false," +
+      "            \"image\": \"quay.io/openshift-release-dev/ocp-release@sha256:e5a6e348721c38a78d9299284fbb5c60fb340135a86b674b038500bf190ad514\"," +
+      "            \"version\": \"4.2.16\"" +
+      "          }" +
+      "        ]," +
+      "        \"conditions\": [" +
+      "          {" +
+      "            \"lastTransitionTime\": \"2020-01-30T04:05:35Z\"," +
+      "            \"status\": \"False\"," +
+      "            \"type\": \"Available\"" +
+      "          }," +
+      "          {" +
+      "            \"lastTransitionTime\": \"2020-01-30T04:31:38Z\"," +
+      "            \"message\": \"Cluster operator monitoring is still updating\"," +
+      "            \"reason\": \"ClusterOperatorNotAvailable\"," +
+      "            \"status\": \"True\"," +
+      "            \"type\": \"Failing\"" +
+      "          }," +
+      "          {" +
+      "            \"lastTransitionTime\": \"2020-01-30T04:05:35Z\"," +
+      "            \"message\": \"Unable to apply 4.2.14: the cluster operator monitoring has not yet successfully rolled out\"," +
+      "            \"reason\": \"ClusterOperatorNotAvailable\"," +
+      "            \"status\": \"True\"," +
+      "            \"type\": \"Progressing\"" +
+      "          }," +
+      "          {" +
+      "            \"lastTransitionTime\": \"2020-01-30T04:05:36Z\"," +
+      "            \"status\": \"True\"," +
+      "            \"type\": \"RetrievedUpdates\"" +
+      "          }" +
+      "        ]," +
+      "        \"desired\": {" +
+      "          \"force\": false," +
+      "          \"image\": \"quay.io/openshift-release-dev/ocp-release@sha256:3fabe939da31f9a31f509251b9f73d321e367aba2d09ff392c2f452f6433a95a\"," +
+      "          \"version\": \"4.8.0-0.nightly-2021-05-26-071911\"" +
+      "        }," +
+      "        \"history\": [" +
+      "          {" +
+      "            \"completionTime\": null," +
+      "            \"image\": \"quay.io/openshift-release-dev/ocp-release@sha256:3fabe939da31f9a31f509251b9f73d321e367aba2d09ff392c2f452f6433a95a\"," +
+      "            \"startedTime\": \"2020-01-30T04:05:35Z\"," +
+      "            \"state\": \"Partial\"," +
+      "            \"verified\": false," +
+      "            \"version\": \"4.8.0-0.nightly-2021-05-26-071911\"" +
+      "          }" +
+      "        ]," +
+      "        \"observedGeneration\": 1," +
+      "        \"versionHash\": \"kEuk9MZ8sK4=\"" +
+      "      }" +
+      "    }" +
+      "  ]," +
+      "  \"kind\": \"ClusterVersionList\"," +
+      "  \"metadata\": {" +
+      "    \"continue\": \"\"," +
+      "    \"resourceVersion\": \"62446\"," +
+      "    \"selfLink\": \"/apis/config.openshift.io/v1/clusterversions\"" +
+      "  }" +
+      "}").once();
```

Comment:
Is it possible to trim parts of response string which are not required for calculating version? Maybe we can move this to `src/test/resources/` and load it from JSON/YAML via `Serialization.jsonMapper().readValue`?